vignettes/ggPMX-guide.Rmd

```diff
@@ -337,7 +399,7 @@ ctr %>% plots()
 
 ### Datasets
 
-`ggPMX` uses the following dataset name convention. The *input modeling dataset* is the one used for model fitting (the actual data). The *output modeling datasets* are those output from the fitting tool (Monolix or NONMEM). The *ggPMX datasets* are the ones created within ggPMX. Table \ref{tab:ggPMX_datasets} provides a list of all ggPMX datasets.
+`ggPMX` uses the following dataset name convention. The *input modeling dataset* is the one used for model fitting (the actual data) (only for Monolix). The *output modeling datasets* are those output from the fitting tool (Monolix,NONMEM or nlmixr). The *ggPMX datasets* are the ones created within ggPMX. Table \ref{tab:ggPMX_datasets} provides a list of all ggPMX datasets.
```

Comment:
Is this comment about monolix for the (input modeling dataset) necessary?  I don't think it adds anything here.

Reply:
It's not necessary here, because it is explained further above. But in generall i would leave it, because it's used throughout the text. Might be changed in a future update, where we could revise more of the text.

For now I added NONMEM and nlmixr, and removed some of the errors